src/libraries/Microsoft.PowerFx.Core/Types/Enums/BuiltInEnums.cs

```diff
@@ -199,5 +199,15 @@ private static Dictionary<string, object> TraceSeverityDictionary()
                 { "IgnoreUnsupportedTypes", TraceFunction.IgnoreUnsupportedTypesEnumValue },
             },
             canConcatenateStronglyTyped: true);
+
+        public static readonly EnumSymbol RemoveFlagsEnum = new EnumSymbol(
+            new DName(LanguageConstants.RemoveFlagsEnumString),
+            DType.String,
+            new Dictionary<string, object>()
+            {
+                { "First", "first" },
+                { "All", "all" },
+            },
+            canCoerceFromBackingKind: true);
```

Comment:
>canCoerceFromBackingKind: true [](http://example.com/codeflow?start=12&length=30)

This should not be coercible - if we use strongly-typed built-in enums, we want makers to write Remove.All, and not "all" .

Reply:
What about other enums that are also coercible (DateTimeFormatEnum)? I unmderstand we want to force makers to to write `Remove.All` but shouldn't we also take in consideration that this is still supported in PA?

Reply:
DateTimeFormat is an enum that is passed to the Text function - and it can also take a string value.